### PR TITLE
Martin/ios 426 fix deeplinks

### PR DESCRIPTION
**Pull request by MartinJibber**
This fixes two things:

1. A crash when tapping on notifications (local or remote).
2. The swift continuation error.

### CODE REVIEW

File: Shared/Managers/LaunchManager.swift [hunk 1]
```diff
@@ -48,9 +48,14 @@ class LaunchManager {
             }))
         }
 
+        // Initialize the stores.
+        UserStore.shared.initialize()
+        await ConnectionStore.shared.initialize()
```
Comment (MartinJibber) [inline Shared/Managers/LaunchManager.swift:53]: Initialize both of these manually. The timing is always after parse has become initialized so we don't error out.

File: Shared/Notifications/UserNotificationManager+Extensions.swift [hunk 2]
```diff
@@ -11,14 +11,21 @@ import UserNotifications
...
+
         return [.banner, .list, .sound, .badge]
     }
 
-    func userNotificationCenter(_ center: UNUserNotificationCenter, didReceive response: UNNotificationResponse) async {
-        
-        if let action = UserNotificationAction.init(rawValue: response.actionIdentifier) {
+    // NOTE: This delegate function seems to get called off the main thread some times which causes a
+    // crash when bringing the app out of the background.
+    @MainActor
```
Comment (MartinJibber) [inline Shared/Notifications/UserNotificationManager+Extensions.swift:24]: For some reason this function was getting called off the main thread which caused a crash.

File: Shared/Stores/ConnectionStore.swift [hunk 3]
```diff
@@ -31,26 +31,21 @@ class ConnectionStore {
         }
     }
 
-    init() {
-        Task {
-            await self.getConnections()
-        }
-    }
```
Comment (MartinJibber) [inline Shared/Stores/ConnectionStore.swift:38]: Get connection was getting called before parse was initialized.

File: Shared/Managers/LaunchManager.swift (hunk 1 above)
Comment (Benjamindodgson) [reply Shared/Managers/LaunchManager.swift:53]: This was causing the error? lame.

File: Shared/Notifications/UserNotificationManager+Extensions.swift (hunk 2 above)
Comment (Benjamindodgson) [reply Shared/Notifications/UserNotificationManager+Extensions.swift:24]: wtf.

**Review by Benjamindodgson — APPROVED** [review]

**Pull request merged**